### PR TITLE
FLPATH-480:recommend option & return workflow type in workflowResponseDto

### PR DESCRIPTION
**Pull request by RichardW98**
**What this PR does / why we need it**:
recommend option & return workflow type in workflowResponseDto
**Which issue(s) this PR fixes**:
Fixes #480

**Change type**
- [ ] New feature
- [x] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [ ] Workflow Service
- [ ] Notification Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java [hunk 1]
```diff
@@ -67,13 +67,6 @@ ProcessAnalysisTask processAnalysisTask(WorkFlowOption move2kube, WorkFlowOption
 				notifier);
 	}
 
-	@Bean
```
Comment (masayag) [inline workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java:70]: not sure if this is the cause of the integration tests failure which complains about unsatisfied dependency - pls have a look at the build log failure.

Comment (RichardW98) [reply workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java:70]: yes the bean dependency 'move2kube' is not loaded in advance

**Pull request merged**